CONTRIBUTING.md

```diff
@@ -67,13 +66,17 @@ Check out the [Go-Filecoin code overview](CODEWALK.md) for a brief tour of the c
 
 ### Design Before Code
 - Write down design intent before writing code, and subject it to constructive feedback.
-- Major changes should have a [Design Doc](https://github.com/filecoin-project/designdocs).
-- For minor changes, file an issue for your change if it doesn't yet have one, and outline your implementation plan on the issue.
+- For a major change, draft a [Design Doc](https://github.com/filecoin-project/designdocs) outlining the context, goals, and your proposed solution.
+- For less a significant change, outline the proposed change in a GitHub issue. If there's already an issue requesting the change, a comment there suffices.
+- Smaller changes, like simple, uncontroversial improvements, don't require an issue. A PR description outlining the motivation and approach is enough. Incremental steps towards a larger outcome don't require individual issues; reference the motivating issue and note each PR achieves.
 
 ### Pull Requests and Reviews
 
-We review every change before merging to master, using GitHub pull requests.
-**Try to keep PRs small**, no more than 400 lines or 8 files. Code reviews are easier, faster, and more effective when the diff is small.
+We review every change before merging to master, using GitHub pull requests. **Try to keep PRs small**, no more than 400 new/changed lines. Code reviews are easier, faster, and more effective when the diff is small.
+ 
+Strive to separate refactors, renames, and significant clean-ups from changes in functionality. 
+Break a large effort into multiple smaller PRs. Each such PR should be self-consistent, well-tested, and a clear step forward, but need not be a complete implementation of a large feature. 
```

Comment:
Something else we could add: if your PR needs to be larger it helps review if it is broken up cleanly into a logical sequence of self contained commits.